src/rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date
+
+  def initialize(date, book, person)
+    @person = person
+    @book = book
+    @date = date
+  end

```

Comment:
- [ ] In order to achieve the `belongs_to` end of the association, the book and the person arguments need to be pushed to their respective `rentals` arrays.

  This can be implemented like so:
  ```
  def initialize(date, book, person)
    @person = person
    person.rentals << self
    @book = book
    book.rentals << self
    @date = date
  end

Reply:
resolved 👍